docs/dev/connectors/kinesis.zh.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Amazon AWS Kinesis Streams Connector"
+title: "Amazon Kinesis Data Streams Connector"
```

Comment:
nit: Would have been nice if these changes were a separate commit (could be a `[hotfix]`), since its non-related to the PR topic at hand.

Reply:
Noted for next time.